### PR TITLE
Auto follow person who referred you upon sign up C-1300

### PR DESCRIPTION
**Pull request by nicoback2**
### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--follow-person-referred

**Review by raymondjacobson — APPROVED** [review]
I think feature flag is probs overkill, but lgtm!

File: packages/web/src/common/store/pages/signon/sagas.js [hunk 1]
```diff
@@ -448,6 +449,17 @@ function* signUp() {
         yield put(signOnActions.signUpSucceeded())
         yield put(signOnActions.sendWelcomeEmail(name))
         yield call(fetchAccountAsync, { isSignUp: true })
+        if (referrer != null) {
```
Comment (dylanjeffers) [inline packages/web/src/common/store/pages/signon/sagas.js:452]: If referrer maybe?

**Review by dylanjeffers — APPROVED** [review]
This is great, love the source being referral

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--follow-person-referred

**Pull request merged**